src/frontend/src/components/EditDialog/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import ConfirmDialog, {
+  ConfirmDialogProps,
+} from "src/components/ConfirmDialog";
+import { StyledSpan } from "src/components/DeleteDialog/style";
+
+interface Props extends Omit<ConfirmDialogProps, "onConfirm"> {
```

Comment:
Nit: Considering how there are now three components based around `ConfirmDialogProps`, this feels to me like we should life out the shared portion into `DialogProps`, then make each of them do their own extends to tack on their one additional prop.